lib/output.js

```diff
@@ -1185,6 +1200,21 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        var parent = output.parent(1);
```

Comment:
Hmm I guess that code is outdated now since I moved the checks elsewhere in the code